### ISSUE TITLE
curl build error

### ISSUE THREAD

**Issue by Doofussy2** [m-ab-s/media-autobuild_suite#2186]
```
Likely error (tail of the failed operation logfile):
../../lib/vtls/mbedtls.c:818:43: error: 'ALPN_H2_LEN' undeclared (first use in this function); did you mean 'ALPN_H2_LENGTH'?
  818 |       if(!strncmp(next_protocol, ALPN_H2, ALPN_H2_LEN) &&
      |                                           ^~~~~~~~~~~
      |                                           ALPN_H2_LENGTH
../../lib/vtls/mbedtls.c:818:43: note: each undeclared identifier is reported only once for each function it appears in
make[2]: *** [Makefile:2952: vtls/libcurl_la-mbedtls.lo] Error 1
make[2]: Leaving directory '/build/curl-git/build-64bit/lib'
make[1]: *** [Makefile:1354: all] Error 2
make[1]: Leaving directory '/build/curl-git/build-64bit/lib'
make: *** [Makefile:1229: all-recursive] Error 1
```

[logs.zip](https://0x0.st/oAyD.zip)

**Comment by Biswa96** [comment]
Duplicate of https://github.com/m-ab-s/media-autobuild_suite/issues/2181. It was fixed in upstream.

**Comment by Doofussy2** [comment]
Ok, thanks. Trying the solution, now.

**Comment by 1480c1** [comment]
Probably need to repull, curl was pinned to the latest tag, and not commit, I unpinned it recently (or just now)

**Comment by Doofussy2** [comment]
I did as the guy in the other thread, did. Deleted those libraries, and it appears to have built successfully.
